### PR TITLE
Consolidated JSP & CSS

### PR DESCRIPTION
**Pull request by Runi-VN**
In regards to issue https://github.com/HrBjarup/Fog-Carport/issues/44
image

Everything seems to work fine.  
**Might have some merge conflicts with current awaiting pull requests!**

## Work done: (see changes)

* Consolidated CSS
* Consolidated JS
* Removed JS/CSS imports and added correct ones
* Added padding on the bottom of the body (all pages)
image

* Made Navbar fixed position at the top, so that it is always displayed. (like the footer)
image
  * Are you happy with this fix? **Discuss!**
* Minor Navbar fixes.
* Added borders to navbar elements (easier to distinguish)
* Removed in-line styling from navbar

## Errors:
* Navbar .active JS doesn't work after the link-change (issue https://github.com/HrBjarup/Fog-Carport/issues/78):
image  
* ~~Still has issue https://github.com/HrBjarup/Fog-Carport/issues/72~~ fixed with merge https://github.com/HrBjarup/Fog-Carport/commit/9546940a31e2c69d7c05ac204110e208e02a200f
* Scrolling navbar has some display issues:
image

#### Drop-down JS works:
image
#### Shed JS _works_:
image

### CODE REVIEW

**Comment by HrBjarup** [comment]
This should probably be merged in after the other pull requests (like you mentioned)

File: FogCarport/src/main/webapp/javascript/global.js [hunk 1]
```diff
@@ -1,4 +1,69 @@
+/*----------------- Navbar JS -----------------  */
```
Comment (MalteMagnussen) [inline FogCarport/src/main/webapp/javascript/global.js]: You may delete this Navbar JS. #78 
It is no longer a viable solution to the class="active" issue.

Comment (MalteMagnussen) [reply FogCarport/src/main/webapp/javascript/global.js]: Just delete the entire navbar js part.

Comment (MalteMagnussen) [reply FogCarport/src/main/webapp/javascript/global.js]: Fixed https://github.com/HrBjarup/Fog-Carport/pull/77/commits/7c0b5083a3adfc174fd815ad3cde9e9f863ae318

**Comment by Runi-VN** [comment]
Deleted navbar javascript
Merge with dev
Checked if https://github.com/HrBjarup/Fog-Carport/pull/80/files was added

**Pull request merged**